extensions/Worker.Extensions.Storage.Blobs/src/Worker.Extensions.Storage.Blobs.csproj

```diff
@@ -7,7 +7,7 @@
 
     <!--Version information-->
     <VersionPrefix>5.1.0</VersionPrefix>
-    <VersionSuffix>-preview1</VersionSuffix>
+    <VersionSuffix>-preview2</VersionSuffix>
```

Comment:
I think we should bump beta to stable release 5.1.0 or 5.1.1

https://github.com/Azure/azure-functions-dotnet-worker/blob/0d49e191d454669fde6186559404df1e2ac1c107/extensions/Worker.Extensions.Storage.Blobs/src/Properties/AssemblyInfo.cs#LL7C34-L7C82

[5.1.2](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/5.1.2)

Reply:
Do you mean, we don't need this in preview now? so 5.1.2 should be the version?

Reply:
No our blob worker ext should stay preview, but we should at least update the assembly info to not point to 5.1.0-beta